### PR TITLE
[IMP] optimised css

### PR DESCRIPTION
**Pull request by sapanz**

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -1,117 +1,116 @@
...
-  -moz-border-radius: 0.3em;
-  border-radius: 0.3em;
+	background: #ecf0f1;
+	color: #34495e;
+	font: normal 1.3em 'Ubuntu', sans-serif;
+	margin: 1em auto;
+	max-width: 45em;
+	padding: 0.3em;
+	-webkit-border-radius: 0.3em;
+	-moz-border-radius: 0.3em;
```
Comment (krman009) [inline styles/style.css]: Remove lines: 21, 22.

File: styles/style.css [hunk 2]
```diff
@@ -1,117 +1,116 @@
...
-  margin-bottom: -0.3em;
-  text-align: center;
-  -webkit-border-radius: 0.3em;
-  -moz-border-radius: 0.3em;
-  border-radius: 0.3em;
+	color: rgba(127, 140, 141, 0.5);
+	margin-bottom: -0.3em;
+	text-align: center;
+	-webkit-border-radius: 0.3em;
+	-moz-border-radius: 0.3em;
```
Comment (krman009) [inline styles/style.css]: Remove lines: 30, 31.

File: styles/style.css [hunk 3]
```diff
@@ -1,117 +1,116 @@
...
-  padding: 0.1em;
-  -webkit-transition: 0.1s all ease-in;
-  -o-transition: 0.1s all ease-in;
-  transition: 0.1s all ease-in;
+	border-bottom: 0.1em solid;
+	color: inherit;
+	text-decoration: none;
+	padding: 0.1em;
+	-webkit-transition: 0.1s all ease-in;
+	-o-transition: 0.1s all ease-in;
```
Comment (krman009) [inline styles/style.css]: Remove lines: 40, 41.

File: styles/style.css [hunk 4]
```diff
@@ -1,117 +1,116 @@
...
-  color: white;
-  padding: 0.2em;
-  text-align: center;
-  -webkit-border-radius: 0.3em;
-  -moz-border-radius: 0.3em;
-  border-radius: 0.3em;
-  border-bottom-left-radius: 0;
-  border-bottom-right-radius: 0;
+	background: #e88630;
+	/*background: #34495e;*/
```
Comment (krman009) [inline styles/style.css]: Remove this comment line.

File: styles/style.css [hunk 5]
```diff
@@ -1,117 +1,116 @@
...
+	color: white;
+	padding: 0.2em;
+	text-align: center;
+	-webkit-border-radius: 0.3em;
+	-moz-border-radius: 0.3em;
```
Comment (krman009) [inline styles/style.css]: Remove lines: 61, 62.

File: styles/style.css [hunk 6]
```diff
@@ -1,117 +1,116 @@
...
 }
 
 .day ul li {
-  background: rgba(52, 73, 94, 0.1);
-  /*border-left: 0.2em solid;*/
-  /*margin: 0.2em;*/
-  padding: 0.37em;
+	background: rgba(52, 73, 94, 0.1);
+	/*border-left: 0.2em solid;*/
+	/*margin: 0.2em;*/
```
Comment (krman009) [inline styles/style.css]: Remove lines containing comment: 84, 85.

**Review by krman009 — CHANGES_REQUESTED** [review]
Please do the changes and indent the changed file using `space` instead of the current one which is `tab`.
Thanks for your work @sapanz :)

**Comment by sapanz** [comment]
Changes done @krman009

**Review by krman009 — APPROVED** [review]
Thank you @sapanz :+1:

**Pull request merged**